specification/powerbidedicated/resource-manager/readme.md

```diff
@@ -34,9 +34,9 @@ tag: package-2017-10-01
 
 These settings apply only when `--tag=package-2017-10-01` is specified on the command line.
 
-``` yaml $(tag) == package-2017-10-01'
+``` yaml $(tag) == 'package-2017-10-01'
 input-file:
-- Microsoft.PowerBIDedicated/package-2017-10-01/powerbidedicated.json
+- Microsoft.PowerBIDedicated/2017-10-01/powerbidedicated.json
```

Comment:
sorry about the delayed reply, I think capitalization is not the same as in the folder structure, therefore reference is not getting properly picked up by CI https://travis-ci.org/Azure/azure-rest-api-specs/jobs/285933140#L617
We're still running the validation on the file, but the link to the readme file, seems not to be working. 
Do you mind double-checking on this?

Reply:
update path in readme file, hope it fixes the issue.
Thanks!